### PR TITLE
Push v0.5.9 to Production

### PR DESCRIPTION
**Pull request by LuckierDodge**
- `wei.core.dataclasses` -> `wei.types.*_types`
- `wei.helpers` -> `wei.utils`
- New `wei.modules.rest_module.RESTModule` class to simplify creating new modules
- Improvements to the experiment client and events system to better support diaspora and pave the way for Datapoints.
- Various bug fixes and performance improvements, primarily around making the Engine/scheduler more stable and performant
- WEI Server now includes a new in-development Graphical User Interface, hosted at the base server url!
- Framework for administrative actions (aka "admin actions") e.g. safety stops, pause, resume, cancel, etc.

### CODE REVIEW

File: tests/test_module/test_rest_node.py [hunk 1]
```diff
@@ -2,235 +2,88 @@
...
-if __name__ == "__main__":
-    import uvicorn
-
-    uvicorn.run(
-        "test_rest_node:app",
-        host=args.host,
-        port=args.port,
-        reload=True,
-    )
+test_rest_node.start()
```
Comment (Dozgulbas) [inline tests/test_module/test_rest_node.py:89]: Might be good to add:

`if __name__ == "__main__":`
`    test_rest_node.start()`

**Review by Dozgulbas — COMMENTED** [review]
Everything looks good to me. Only added minor comments

File: .github/workflows/docker-publish.yml [hunk 2]
```diff
@@ -61,7 +62,7 @@ jobs:
             type=ref,event=pr
             type=raw,value=latest,enable={{is_default_branch}}
 
-      # Build and push Docker image with Buildx (don't push on PR)
```
[the remark this answers is not in the document]
Comment (LuckierDodge) [reply .github/workflows/docker-publish.yml:64]: No longer relevant, we used to have two separate workflows, one for PRs and one for the main branch and releases, but we've simplified

File: compose.yaml [hunk 3]
```diff
@@ -72,26 +42,29 @@ services:
...
-      - ${WEI_DATA_DIR}:/home/app/.wei
       - diaspora_config:/home/app/.diaspora
-      - ${PROJECT_DIR}:/home/app/wei # for development only
-    env_file:
-      - .env
+      - ./tests/workcells:/workcell_defs
+      - ~/.wei:/home/app/.wei
+
+    environment:
+      - PYTHONUNBUFFERED=1 # Fix weird bug with empty logging
```
[the remark this answers is not in the document]
Comment (LuckierDodge) [reply compose.yaml:50]: This environment variable is the fix, basically. It's a problem with python<->docker interactions, not something on our end.

File: pyproject.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [project]
 name = "ad_sdl.wei"
-version = "0.5.8"
+version = "0.5.9"
 description = "The Rapid Prototyping Laboratory's Workflow Execution Interface."
 authors = [
```
[the remark this answers is not in the document]
Comment (LuckierDodge) [reply pyproject.toml:5]: Is the current list, on lines 6-11, missing someone?

[the remark this answers is not in the document]
Comment (Dozgulbas) [reply pyproject.toml:5]: Discard this

**Review by Dozgulbas — APPROVED** [review]

**Pull request merged**